src/pybind/mgr/dashboard/controllers/docs.py

```diff
@@ -169,7 +167,7 @@ def _gen_schema_for_content(cls, params):
             else:
                 props['type'] = cls._gen_type(param)
             if 'description' in param:
-                props['description'] = param['description']
+                props['description'] = param['description'] or "*No Description Available*"
```

Comment:
Can this be the same as the previous one, or do we need 2 different texts for them?

Reply:
I can revert it; we have a method that fixes this now, I suspect, with that none function.